app/Providers/RouteServiceProvider.php

```diff
@@ -53,6 +53,7 @@ public function map()
         $this->mapPartnerRoutes();
         $this->mapAuthRoutes();
         $this->mapLocalessRoutes();
+        $this->mapGoogle2FARoutes();
```

Comment:
Personally I think these routes can live in mapProfileRoutes, as they are linked to the user's profile data.